e2-studio-star-rx72n-firmware/libs/rx_crc/src/rx_dmaca.c

```
@@ -0,0 +1,314 @@
+     */
+    if ((p_cfg->data_size != k_dmaca_size_byte) &&
+        (p_cfg->data_size != k_dmaca_size_longword)) {
+        return k_rx_err_param;
+    }
+
+    if ((p_cfg->p_src == NULL) || (p_cfg->p_dst == NULL)) {
+        return k_rx_err_param;
+    }
+
+    /* ---- Stop the channel before reconfiguring -------------------------- */
+    volatile struct st_dmac0 *ch = dmac_ch(channel);
```

Clang (14.0.6)

[warning] variable 'ch' is not initialized

(cppcoreguidelines-init-variables)